engine/src/cmd/missile.h

```diff
@@ -78,14 +80,6 @@ class GameMissile : public GameUnit< Missile >
                                              ResolveLast,
                                              uc );
     }
-
-private:
-/// default constructor forbidden
-    GameMissile();
-/// copy constructor forbidden
-    GameMissile( const Missile& );
-/// assignment operator forbidden
-    GameMissile& operator=( const Missile& );
```

Comment:
Hmm. We are no longer forbidding these constructors? I'll have to think about that.

Reply:
@royfalk I suspect that there was a reason why the previous developer(s) forbade these constructors. Can you please put this code back in? Thanks!

Reply:
You are absolutely right. I did not fully grasp the purpose of what they did here. Returned the code for now.